backend/routes/api/resources.js

```diff
@@ -0,0 +1,151 @@
+const express = require("express");
+const router = express.Router();
+const Resource = require("../../models/Resource");
+const { celebrate, Joi } = require("celebrate");
+
+// get all resources
+router.get("/", async (req, res) => {
+  const resources = await Resource.find({});
+  res.json({
+    code: 200,
+    result: resources,
+    success: true
+  });
+});
+
+// get one resource
+router.get("/:resource_id", async (req, res) => {
+  const resourceId = req.params.resource_id;
+  const resource = await Resource.findById(resourceId);
+  res.json({
+    code: 200,
+    result: resource,
+    success: true
+  });
+});
+
+// get list of resources filtered by location radius
+router.get("/filter", async (req, res) => {
+  const radius = req.query.radius;
+  const lat = parseInt(req.query.lat);
+  const long = parseInt(req.query.long);
+  const resource = await Resource.find({
+    location: { $geoWithin: { $centerSphere: [[long, lat], radius / 3963.2] } }
```

Comment:
Awesome! If you can just add a quick comment above describing why `3963.2` division, that'd be great.